native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -77,7 +77,7 @@ const styles = StyleSheet.create({
     color: colorGray700,
     fontWeight: textLgFontWeight,
     // TODO Replace with fontFamilyLg token once available
```

Comment:
is the todo still relevant?

Reply:
Nope, I thought I removed that

Reply:
never mind, I submitted the review while you were pushing 😅